kibana/values.yaml

```diff
@@ -72,7 +72,32 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 1000
 
-serviceAccount: ""
+# Whether this chart should self-manage its service account, role, and associated role binding.
+managedServiceAccount: true
+
+rbac:
+  create: false
+  serviceAccountName: ""
+
+podSecurityPolicy:
+  create: false
+  name: ""
+  spec:
+    privileged: true
+    fsGroup:
+      rule: RunAsAny
+    runAsUser:
+      rule: RunAsAny
+    seLinux:
+      rule: RunAsAny
+    supplementalGroups:
+      rule: RunAsAny
+    volumes:
```

Comment:
Let's add `downwardAPI` and `emptyDir` like in https://github.com/elastic/helm-charts/pull/496#discussion_r440934948